egui_demo_lib/src/apps/demo/tests.rs

```diff
@@ -1,3 +1,5 @@
+use egui::Rgba;
```

Comment:
please no top-level `use` statements in the demo lib - I try to be as explicit as possible to make it easier for people to copy-paste the code they want